README.md

```diff
@@ -29,7 +29,7 @@ You can create a PR with your draft article (or edits on an existing article) in
 
 1) The easiest method is to use the GitHub interface. Watch the video demonstration or follow the steps below it:
 
-![Gif showing the GitHub interface steps](https://files.gitter.im/FreeCodeCamp/building-guides/WPK4/guide-article-contribution.gif)
+![Gif showing the GitHub interface steps](https://i.imgur.com/FomDGuQ.gif)
```

Comment:
@shovanch thanks for for taking the time to do this. Changes I would like to see are: 

Can you do the edit on the freeCodeCamp/guides repo, not your fork. You can select the option the commit the changes to your fork.

Either drop every other frame when compiling the gif, or speed up the actions you take. The gif constantly repeats so we shouldn't be too concerned about people picking it all up on the first pass

Make the changes more drastic, like delete the whole stub and write 'my changes' then commit.

Reply:
Sure. Would get this done

Reply:
@Bouncey Sorry for the delay. I have made a updated GIF as requested. Please take a look. If it's OK then I will go ahead and commit it. 
GIF link: https://i.imgur.com/0cmxJwN.gif